modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,30 +5,112 @@
+This section describes how to set up Aura SSO with Role-Based Access Control (RBAC) for centrally controlled privileges as well as data access.
+
+Aura Enterprise supports SSO at the organization, project, and instance levels.
+Aura admins can configure organization and project level SSO while Aura Support can configure instance level SSO.
+
+== Requirements
 
-[NOTE]
-====
 Accessing Aura with SSO requires:
  
-* Authorization Code Flow with PKCE.
+* Authorization Code Flow with PKCE (Proof Key for Code Exchange).
 * A publicly accessible Identity Provider (IdP) server.
-====
+
+== Configuring SSO 
+
+Users can configure a combination of one or more of the following login methods:
+
+* Email/password
+* Okta
+* Microsoft Entra ID (previously Azure Active Directory)
+
+An organization's admin can add Aura as a login from a tile in an organization's Apps Dashboard.
+
+
+== SSO setup requirements 
+
+To configure SSO from Aura, go to the *Aura Console > Settings > SSO Configuration.*
+
+The set-up form requires the following info:
+
+* SSO config ID - you decide a name for this
+* IdP type (e.g. Okta / Microsoft Entra ID)
+* Channel
+* Client ID 
+* Client Secret
+* Discovery URI
+* Issuer
+* Authorization Endpoint
+
+== Organization SSO
+
+Organization SSO allows log in to the Aura Console using organization IdP credentials.
+
+To facilitate organization SSO login, you need to create an Auth0 organization and add all users to it.
+Then you need to associate tenants/projects with an organization and add the users to it when they accept invitations to join.
+
+== Mapping an organization
+
+When you map an organization to your IdP, Aura grants membership in that organization to users who authenticate through the IdP .
+
+You can give these users a default role in the selected organization.
+
+Organization mapping lets you configure a single IdP to grant users access to multiple organizations within the Neo4j Aura Console. 
+
+.Organization roles
+[cols="25m,75v"]
+|===
+| Role | Permissions
+
+| Organization Owner
+| • Full access to read/write organization
+
+• Full access to list/read/write tenants
+
+• Given to self serve users during registration instead of a tenant role
+
+• Can create/edit/delete SSO [Enterprise Only]
+
+• Can invite to any role up to owner
+
+| Organization Admin
+| • Full access to read/write org details
+
+• Full access to list/create tenants
+
+• Can create/edit/delete SSO [Enterprise Only]
+
+• Can invite to any role up to admin
+
+| Organization Member (Default when added to tenant)
+| • Full access to read organization
+
+• Full access to list tenants
+
+• Default role for users given access to a single tenant via invite/marketplace
+
+|===
+
+== Home Realm Discovery 
+
+If Home Realm Discovery (HRD) is enabled, and a user’s email domain matches the configured domain, they are redirected to the Auth0 connection IdP. 
```

Comment:
We don't need to mention Auth0 to the customer i don't beleive. Ideally they know nothing at all about auth0. They are just redirected from our login page to their IDP's login page.